src/components/NavBar/Links.js

```diff
@@ -45,7 +45,15 @@ const Link = styled(LinkItem)`
   }
 
   /* stylelint-disable */
-  &:focus,
+  &:focus {
+    outline: 1px solid ${themes.global.white.base};
+    .nav--inverted &,
+    .links__list & {
+      background-color: ${colors.shale};
+      outline: 1px solid ${themes.global.primary.base};
+    }
+  }
```

Comment:
Hm.. Is it a hard requirement to keep the outline style as a focus indicator? My understanding is that having the tinted background should serve this purpose? 

Anyway, we could just rely on the default UA focus indicator as WCAG 2.0. suggests. I.e. we would need to remove the style on line #50 in the original code. And final styles could look like:

```
&:focus,
&:hover {
   background-color: rgba(38, 38, 38, 0.1);
    .nav--inverted &,
    .links__list & {
      background-color: ${colors.shale};
    }
}
```

Reply:
Thank you for your feedback. While it's certainly not a hard requirement to have an outline as a focus indicator. The documentation and WebAIM suggest that we have a visible outline for focus: https://webaim.org/techniques/keyboard/#problems and https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html under the `Examples of Success Criterion 2.4.7` section. It's also implemented in the EDP as well: https://www.ticketmaster.com/harry-styles-love-on-tour-inglewood-california-08-30-2021/event/0900577009DB6328 (you can navigate by tabbing and will see an outline as the focus indicator but not on hover. The EDP current accessibility features have passed the WebAIM audit last year. Moreover, having only a tinted background is not enough because, in the Event Lading Page project, we have a dark background in the NavBar and it's really hard to see the styling change when hover and focus.

In addition, the changes that I made to separate out the focus outline style is because I don't want to touch the current implementation of the hover state. 

I've also tried the suggested block above but it doesn't work as expected because I couldn't see any outline when focus and hover.